### DISCUSSION TITLE
Updates for MATLAB Release R2023a

### DISCUSSION THREAD

**Discussion by mmcbroom** [nasa/cFS#687] [Q&A]
Hello.  This is Mark McBroom from MathWorks.  Walt Stuart from team worked with NASA to develop the cFS capability.  Walt is no longer supporting this effort, but another member of my team Pratheek Gowda (github ID: Gowda-Pratheek) made some minor code updates to support newer releases of MATLAB through R2023a.  Would you like us to provide those changes to NASA?  

Thanks.
Mark McBroom
Application Engineering Manager
MathWorks

**Comment by justinduppy** [comment]
Hi Mark, 

Justin here. Not NASA affiliated but me and my team were planning on attempting to integrate some auto generated code from Simulink into cFS using the Simulink Interface Layer (SIL) and R2023a in just a couple weeks from now so thought I'd chime in. 

I was certainly expecting some small compatability changes required but if your colleague has already done the hard work it would be super helpful if they were made available! Perhaps on a seperate fork if the cFS maintenance team is slow to respond?

In any case, great to hear some guys on this inside are still playing around with this stuff and thanks to you and Pratheek for the work and pro-activity. 

Regards,
Justin du Plessis
